log-window.js

```diff
@@ -158,6 +170,23 @@ class EmeLoggerWindow {
     return '0x' + byte.toString(16).padStart(2, '0');
   }
 
+  /**
+   * @param {number} byte
+   * @return {string}
+   * @private
+   */
+  bytesToBase64_(bytes) {
+    return btoa(String.fromCharCode.apply(null,new Uint8Array(bytes)));
```

Comment:
nit: Reformating seems to be needed for this file (i.e., space after comma). You may want to set up automatic running clang-format on Save if you are using VSCode.

Reply:
done.